app/controllers/course_instances_controller.rb

```diff
@@ -22,6 +20,8 @@ def autocomplete
   private
 
   def autocomplete_label(meeting)
+    # call .strip on the schedule because some of the imported
+    # data from SIS has trailing whitespace
```

Comment:
Perhaps we should deal with the trailing whitespace at the import level?
And we can always just run a quick command in the console to address already imported data.